### PR TITLE
add terms of modification

### LINKED ISSUE #82: Different licence for the licence?
I think it might be beneficial if the licence text was put under a permissive licence so that it can be used for derivative licences without having to include licence notices in the derivative. I think that would be acceptable since a licence text can't cause much harm and the “evil guys” targeted by it wouldn't create a derivative licence anyway.

### PR DESCRIPTION
**Pull request by ghost**
## Overview
Fixes #82.

## Proposed Resolution
Adds terms of modification to the license.

### CODE REVIEW

**Comment by realpixelcode** [comment]
Oh, I think we misunderstood each other a bit: What I meant was to replace the licence for the entire repo with a Creative Commons licence and to put the modification clause into that same CC licence.

Putting the modification clause into the Do No Harm Licence itself is a bit recursive, I guess 😅

**Comment by ghost** [comment]
Ok, that sounds good. ~~however, according to Creative Commons, modification of a CC license is not a good idea.~~

**Comment by ghost** [comment]
Let me remove that change, add a CC0 license, and add the terns of modification clause.

**Comment by ghost** [comment]
Scratch that; the legal code *will* have to be linked to from CC's website, and the terms of distribution will need to be in the README. The only thing that could be downloaded is the CC0 [plaque/image](https://creativecommons.org/about/downloads/).

**Comment by realpixelcode** [comment]
Okay, that's a good alternative! Maybe remove the `LICENSE.md` to avoid confusion?

**Comment by realpixelcode** [comment]
Oh, I think we forgot one thing:

> If copied verbatim without any modifications, the Do No Harm Licence may be used and re-distributed according to CC0 without any further restrictions (like having to change the title).

**Comment by ghost** [comment]
Done. What do you think of the comments in the review I started?

**Comment by realpixelcode** [comment]
Sorry, I don't see any. For me, it says `Reviews: None requested` 🤔

**Comment by ghost** [comment]
Huh, it should say "**IRod22** started a review" on the conversation panel.

**Comment by realpixelcode** [comment]
Maybe only the project owner can view it? (It's neither visible in the app nor on the website 🤔)

**Comment by ghost** [comment]
Huh, that's weird. What I meant to say is that line 63 may look a bit confusing for a dev to understand.
```markdown
Provided that your derivative of this licence
 * does not imply directly or indirectly that such a derivative is supported by the original licence creators or users, <!-- [?] discuss -->
```
After all, we are making the terms of modification of the license more approachable to developers, not legal experts.

**Comment by realpixelcode** [comment]
You mean we should simplify the wording? How about something like this?

> You may use the content of the Do No Harm Licence according to the [CC0 License](https://creativecommons.org/publicdomain/zero/1.0/legalcode) (read the [summary](https://creativecommons.org/publicdomain/zero/1.0/)), but if you modify the content, you must also change the name of your derivative licence in order to prevent any confusion. Also, do not claim that we, the original licence creators, would endorse your specific derivative.

**Comment by ghost** [comment]
Sorry for the late response, @realpixelcode. Let me add that to the `README.md`.

**Comment by ghost** [comment]
I meant to say `simplify terms of modification` in the commit message.

---

Alright. @tommaitland, this PR is ready for merging.

**Comment by MrAureliusR** [comment]
As someone who made the suggestion, I was definitely intending the sentence regarding derivatives to be put IN the text of the license itself. I believe this is how most other Licenses do it. I think it's best to have it all in one document. Again, I really don't understand why we are licensing the license itself. I suppose it can't do too much harm, but it just seems like a future stumbling block.

Also, license proliferation is a real issue. The current wording sounds like we are encouraging people to make derivatives of the license if they want to add clauses; when really they should either propose changes to this license or try to find one that already meets their needs. There are a huge number of stagnant licenses out there that are genuinely causing issues in open source software. I would again argue that this should be worded in a way that gently discourages license proliferation, while also being firm that derivatives must be named differently.

**Comment by ghost** [comment]
@MrAureliusR, you make a good point; before we merge this PR, license proliferation will need more discussion with @tommaitland, the rest of the core team, and other contributors. That was why I considered CC BY-** as a more ideal license. @realpixelcode, what do you think?
